wordpress-mobile/WordPress-Android
# Post Settings: Password

This is a small PR which updates the Post Settings: Password field to the latest design.

To test:
* Go into Post Settings and observe different scenarios where you update the password
